.github/workflows/pythontest.yml

```diff
@@ -9,22 +9,23 @@ jobs:
       max-parallel: 4
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: ["3.7", "3.8", "3.9"]
```

Comment:
Why not add 3.10 to the test?

Reply:
I don't want to extend the code change.
That's the reason.
But if you think it is required, then I will add 3.10

Reply:
Thanks 🙇